src/utils/swarm-session-id.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * Swarm Session ID operations on URLs
+ */
+export const SWARM_SESSION_ID_KEY = 'swarm-session-id'
+
+/**
+ * It removes the Swarm Session ID query parameter from the URL
+ *
+ * Swarm Session ID has to be removed from the URL so that the ID won't be exposed.
+ *
+ * @param bzzUrl BZZ URL with arbitrary query parameters, e.g. http://.../1231abcd.../valami.html?swarm-session-id=vmi&smth=5
+ * @returns url without the SwarmSessionID
+ */
+export function removeSwarmSessionIdFromURL(bzzUrl: string): string {
```

Comment:
Please change `URL` to `Url` for correct camelCase.

Other function is correct: `getSwarmSessionIdFromUrl`.